### DISCUSSION TITLE
Post Title is on the right side

### DISCUSSION THREAD

**Discussion by ghost** [hugo-fixit/FixIt#379] [Q&A] [answered]
How do i allign it to the left side?
![grafik](https://github.com/hugo-fixit/FixIt/assets/124097050/3f184635-f94b-4201-868f-804e955060e4)

Thanks in advance.

**Comment by Lruihao** [comment, marked as answer]
Place articles in the posts folder or set frontmatter `type: posts`.

**Comment by ipfred** [comment]
所有的md文件都必须放在post下面吗

**Reply by Lruihao** [reply]
放在 posts 文件夹，或者front matter 加上 type: posts
